lib/ansible/module_utils/common/process.py

```diff
@@ -6,19 +6,27 @@
 import os
 
 from ansible.module_utils.common.file import is_executable
+from ansible.module_utils.common.warnings import deprecate
 
 
 def get_bin_path(arg, opt_dirs=None, required=None):
     '''
-    Find system executable in PATH. Raises ValueError if executable is not found.
+    Find system executable in PATH. Raises ValueError if the executable is not found.
     Optional arguments:
-       - required:  [Deprecated] Prior to 2.10, if executable is not found and required is true it raises an Exception.
-                    In 2.10 and later, an Exception is always raised. This parameter will be removed in 2.14.
+       - required:  [Deprecated] Before 2.10, if executable is not found and required is true it raises an Exception.
+                    In 2.10 and later, an Exception is always raised. This parameter will be removed in 2.21.
        - opt_dirs:  optional list of directories to search in addition to PATH
     In addition to PATH and opt_dirs, this function also looks through /sbin, /usr/sbin and /usr/local/sbin. A lot of
     modules, especially for gathering facts, depend on this behaviour.
     If found return full path, otherwise raise ValueError.
     '''
+    if required:
```

Comment:
`if required is not None` (in case someone is passing `False`.

Reply:
True